src/Approvals/SFA.DAS.Approvals/Services/FjaaService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Approvals.InnerApi.Requests;
+using SFA.DAS.Approvals.InnerApi.Responses;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Extensions;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Approvals.Services
+{
+    public interface IFjaaService
+    {
+        Task<bool> IsLegalEntityOnFjaaRegister(long accountLegalEntityId);
+    }
+
+    public class FjaaService : IFjaaService
+    {
+        private readonly IFjaaApiClient<FjaaApiConfiguration> _fjaaClient;
+        private readonly ICommitmentsV2ApiClient<CommitmentsV2ApiConfiguration> _commitmentsV2ApiClient;
+        private readonly ILogger<DeliveryModelService> _logger;
+
+        public FjaaService(IFjaaApiClient<FjaaApiConfiguration> fjaaClient, ICommitmentsV2ApiClient<CommitmentsV2ApiConfiguration> commitmentsV2ApiClient, ILogger<DeliveryModelService> logger)
+        {
+            _fjaaClient = fjaaClient;
+            _commitmentsV2ApiClient = commitmentsV2ApiClient;
+            _logger = logger;
+        }
+
+        public async Task<bool> IsLegalEntityOnFjaaRegister(long accountLegalEntityId)
+        {
+            if (accountLegalEntityId == 0) return false;
```

Comment:
Is this a good time to actually remove this line? As I recall it was just there to allow it to work prior to UI changes being deployed.

Reply:
Agreed, we did, removed